dojo/tools/dependency_check/parser.py

```diff
@@ -238,6 +238,10 @@ def get_finding_from_vulnerability(self, dependency, related_dependency, vulnera
                                      'source: {1}\n' \
                                      'url: {2}\n\n'.format(name, source, url)
 
+        component_name, component_version = self.get_component_name_and_version_from_dependency(dependency, related_dependency, namespace)
+
```

Comment:
don't think this is needed/ intended?

Reply:
Yup, it was not intended, thanks for catching that